src/components/molecules/menu/menu.tsx

```diff
@@ -88,39 +67,47 @@ export const Menu = ({ className, isOpen = false, onMouseLeave, onSubmenu }: Men
 						</AnimatePresence>
 					</Button>
 				</li>
-
-				<li
-					className="group static"
-					onMouseEnter={(event) => handleMouseEnter(event, sortedProjectsList)}
-					onMouseLeave={onMouseLeave}
-				>
-					<div className="cursor-pointer before:absolute before:left-0 before:h-10 before:w-full" />
-
-					<Button ariaLabel={t("myProjects")} className={buttonMenuStyle("#")} title={t("myProjects")}>
-						<div className={buttonMenuIconWrapperStyle("#")}>
-							<IconSvg
-								alt={t("myProjects")}
-								className={buttonMenuIconStyle("#")}
-								size="xl"
-								src={ProjectsIcon}
-							/>
-						</div>
-
-						<AnimatePresence>
-							{isOpen ? (
-								<motion.span
-									animate="visible"
-									className="overflow-hidden whitespace-nowrap"
-									exit="hidden"
-									initial="hidden"
-									variants={animateVariant}
-								>
-									{t("myProjects")}
-								</motion.span>
-							) : null}
-						</AnimatePresence>
-					</Button>
-				</li>
+				<PopoverList animation="slideFromLeft">
+					<PopoverListTrigger>
+						<li className="group static">
+							<div className="cursor-pointer before:absolute before:left-0 before:h-10 before:w-full" />
+							<div className="relative z-10 w-full gap-1.5 rounded-full p-0.5 pl-1 group-hover:bg-green-200">
+								<div className="flex size-9 items-center justify-center rounded-full">
+									<IconSvg
+										alt={t("myProjects")}
+										className="fill-gray-1100"
+										size="xl"
+										src={ProjectsIcon}
+									/>
+								</div>
+
+								<AnimatePresence>
+									{isOpen ? (
+										<motion.span
+											animate="visible"
+											className="overflow-hidden whitespace-nowrap"
+											exit="hidden"
+											initial="hidden"
+											variants={animateVariant}
+										>
+											{t("myProjects")}
+										</motion.span>
+									) : null}
+								</AnimatePresence>
+							</div>
+						</li>
+					</PopoverListTrigger>
+					<PopoverListContent
+						className="z-30 flex h-screen flex-col rounded-lg bg-white px-4 pt-[212px] text-black"
+						itemClassName={cn(
+							"flex cursor-pointer items-center gap-2.5 rounded-3xl p-2 transition",
+							"hover:text-current text-center text-gray-1100 duration-300 hover:bg-gray-1250",
+							"text-fira-code whitespace-nowrap px-4 text-gray-1100 hover:bg-green-200 max-w-245 overflow-hidden"
+						)}
+						items={sortedProjectsList.map(({ id, name }) => ({ id, label: name, value: id }))}
+						onItemSelect={({ id: projectId }) => navigate(`/${SidebarHrefMenu.projects}/${projectId}/code`)}
```

Comment:
if this list will be used only in the sidebar. This prop doesn't need

Reply:
What do you mean? this is practically our `onClick`

Reply:
all right